backend/src/api/app_factory.py

```
@@ -139,7 +148,10 @@ def _configure_routes(app: FastAPI) -> None:
     """Configure application routes"""
 
     # Import route modules
-    from .routes import upload, status, enhanced_guide, preferences, generation, pdf, pdf_html, health, debug, places
+    from .routes import (
+        upload, status, enhanced_guide, preferences, generation, pdf,
```

Ruff (0.13.1)

Redefinition of unused `upload` from line 13

Remove definition: `upload`

(F811)